### ISSUE TITLE
unsupported operand type(s) for *: 'NoneType' and 'int'

### ISSUE THREAD

**Issue by Richardzhouzherui** [pythongosssss/ComfyUI-WD14-Tagger#57]
Error occurred when executing WD14Tagger|pysssss:

unsupported operand type(s) for *: 'NoneType' and 'int'

File "D:\ComfyUi\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUi\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUi\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUi\custom_nodes\ComfyUI-WD14-Tagger\wd14tagger.py", line 193, in tag
tensor = image*255

**Comment by sapientsnap** [comment]
Same problem. Reinstalled WD1.4 node. Dead in the water. :(

**Comment by vnguyen972** [comment]
yeah same here... how to resolve this? I tested with the same tagger model specified here with a sample image and its working fine... but the node keeps failing with this error.
